react/components/ProductName/index.js

```diff
@@ -25,19 +24,31 @@ class ProductName extends Component {
     showBrandName: PropTypes.bool,
     /** Display large font */
     large: PropTypes.bool,
+    /** Extra classes for loader of the component */
+    loaderClasses: PropTypes.string,
+    /** Extra classes for root of the component */
+    rootClasses: PropTypes.string,
     /** Component and content loader styles */
     styles: PropTypes.object,
+    /** Render function for name field, this function should expect a name string and large prop */
+    renderName: PropTypes.func,
+    /** Render function for sku field, this function should expect a skuName string and large prop */
+    renderSku: PropTypes.func,
+    /** Render function for product reference field, this function should expect a productReference string and large prop */
+    renderProductReference: PropTypes.func
   }
 
   static defaultProps = {
     large: false,
     showBrandName: false,
     showProductReference: false,
     showSku: false,
+    loaderClasses: '',
+    rootClasses: ''
```

Comment:
no default for the render methods?

Reply:
I mean, for a `ProductName` component I would expect it to at least render a span with the name I passed... I don't know if you already discussed it with @brunojdo

Reply:
You mean like
```javascript
// code...
static defaultProps = {
  //...
  renderName: ({ name }) => <span>{name}</span>
}
```
?